### PR TITLE
Update Migration Docs

### PR DESCRIPTION
**Pull request by isharailanga**
This PR is to update the Migration docs.
- Fix https://github.com/wso2/docs-open-banking/issues/548#issuecomment-1125707064. 
- Add steps to configure deployment.toml files.

### CODE REVIEW

File: en/docs/install-and-setup/upgrading-the-solution/reporting-data-migration.md [hunk 1]
```diff
@@ -36,11 +36,14 @@ as follows:
...
 5. Replace the existing `deployment.yaml` file in the Streaming Integrator as follows:
-   - Go to the `<SI_HOME>/<OB_BI_ACCELERATOR_HOME>/repository/resources` directory.
-   - Rename `wso2si-4.0.0-deployment.yaml` to `deployment.yaml`.
-   - Copy the `deployment.yaml` file to the `<SI_HOME>/conf/server` directory to replace the existing file.
+    - Go to the `<SI_HOME>/<OB_BI_ACCELERATOR_HOME>/repository/resources` directory.
+    - Rename `wso2si-4.0.0-deployment.yaml` to `deployment.yaml`.
+    - Copy the `deployment.yaml` file to the `<SI_HOME>/conf/server` directory to replace the existing file.
+
+6. Open the `<SI_HOME>/conf/server/deployment.toml` file, and configure the hostnames and databases related
+   properties accordingly.
```
Comment (malshaniS) [inline en/docs/install-and-setup/upgrading-the-solution/reporting-data-migration.md:44]: Let's mention to point the databases creates for Open Banking 2.0.

File: en/docs/install-and-setup/upgrading-the-solution/upgrading-wso2-api-manager-400.md [hunk 2]
```diff
@@ -32,19 +32,22 @@
     - Go to the `<APIM_HOME>/<OB_APIM_TOOLKIT_HOME>/repository/resources` directory.
     - Rename `wso2am-4.0.0-deployment-uk.toml` to `deployment.toml`.
     - Copy the `deployment.toml` file to the `<APIM_HOME>/repository/conf` directory and replace the existing file.
+    
+ 10. Open the `<APIM_HOME>/repository/conf/deployment.toml` file, and configure the hostnames and databases related
```
Comment (malshaniS) [inline en/docs/install-and-setup/upgrading-the-solution/upgrading-wso2-api-manager-400.md:36]: Let's mention to point the databases creates for Open Banking 2.0.

File: en/docs/install-and-setup/upgrading-the-solution/upgrading-wso2-identity-server.md [hunk 3]
```diff
@@ -41,6 +41,9 @@
     - Rename `wso2is-5.11.0-deployment-uk.toml` to `deployment.toml`.
     - Copy the `deployment.toml` file to the `<IS_HOME>/repository/conf` directory to replace the existing file.
 
+12. Open the `<IS_HOME>/repository/conf/deployment.toml` file, and configure the hostnames and databases related
```
Comment (malshaniS) [inline en/docs/install-and-setup/upgrading-the-solution/upgrading-wso2-identity-server.md:44]: Let's mention to point the databases creates for Open Banking 2.0.

**Review by malshaniS — CHANGES_REQUESTED** [review]

File: en/docs/install-and-setup/upgrading-the-solution/reporting-data-migration.md (hunk 1 above)
Comment (isharailanga) [reply en/docs/install-and-setup/upgrading-the-solution/reporting-data-migration.md]: Fixed in f9ac8569d3756f0ee97e73443ce1c7ea47c0aa5b

File: en/docs/install-and-setup/upgrading-the-solution/upgrading-wso2-api-manager-400.md (hunk 2 above)
Comment (isharailanga) [reply en/docs/install-and-setup/upgrading-the-solution/upgrading-wso2-api-manager-400.md]: Fixed in f9ac8569d3756f0ee97e73443ce1c7ea47c0aa5b

**Review by malshaniS — APPROVED** [review]

**Pull request merged**